Editor/Utilities/EditorUtils.cs

```diff
@@ -24,5 +26,167 @@ public static void OpenPropertyEditor(UnityEngine.Object obj)
             openPropertyEditorMethodInfo?.Invoke(null, new object[] { obj, true });
 #endif
         }
+        
+        /// <summary>
+        /// Gets the object that the property is a member of
+        /// </summary>
+        /// <param name="property"></param>
+        /// <returns></returns>
+        public static object GetTargetObjectWithProperty(SerializedProperty property)
+        {
+            string path = property.propertyPath.Replace(".Array.data[", "[");
```

Comment:
Maybe do property.serializedObject.ApplyModifiedProperties(); and property.serializedObject.Update(); first so that target object is always up to date.